### PR TITLE
Plat 2.6 & sbt update.

### PR DESCRIPTION
**Pull request by rhubner**
I also increase `.settings(majorVersion := 1)`
We can easy see which version of appplication is with play 2.5 and 2.6.

### CODE REVIEW

File: app/uk/gov/hmrc/valuationofficeagencycontactfrontend/filters/SessionIdFilter.scala [hunk 1]
```diff
@@ -44,38 +45,23 @@ class SessionIdFilter (
...
-        val session: Session =
-          rh.session + (SessionKeys.sessionId -> sessionId)
-
-        val cookies =
-          rh.cookies ++ Seq(Session.encodeAsCookie(session))
-
-        Cookies.encodeCookieHeader(cookies.toSeq)
-      }
+      val session: Session =
+        rh.session + (SessionKeys.sessionId -> sessionId)
```
Comment (valydia) [inline app/uk/gov/hmrc/valuationofficeagencycontactfrontend/filters/SessionIdFilter.scala:49]: Aren't the cookies needed anymore?

File: project/Dependencies.scala [hunk 2]
```diff
@@ -30,13 +31,18 @@ object Dependencies {
...
+      "uk.gov.hmrc" %% "bootstrap-play-26" % bootstrapPlayVersion,
       "uk.gov.hmrc" %% "play-language" % playLanguageVersion,
-      "uk.gov.hmrc" %% "http-verbs" % "10.7.0-play-25"
+      "uk.gov.hmrc" %% "http-verbs" % "10.7.0-play-26"
     )
 
-    lazy val appDependencyOverrides: Set[ModuleID] = Set(
-      "uk.gov.hmrc" %% "govuk-template" % govukTemplateVersion
+    lazy val appDependencyOverrides: Seq[ModuleID] = Seq(
+//      "uk.gov.hmrc" %% "govuk-template" % govukTemplateVersion,
```
Comment (valydia) [inline project/Dependencies.scala:40]: Any reason to keep this commented out line ?

File: app/uk/gov/hmrc/valuationofficeagencycontactfrontend/filters/SessionIdFilter.scala (hunk 1 above)
Comment (rhubner) [reply app/uk/gov/hmrc/valuationofficeagencycontactfrontend/filters/SessionIdFilter.scala:49]: We have same solution on RALD. I didn't want to go to much deep.

File: test/uk/gov/hmrc/valuationofficeagencycontactfrontend/views/ConfirmationViewSpec.scala [hunk 3]
```diff
@@ -26,13 +26,13 @@ class ConfirmationViewSpec extends ViewBehaviours {
 
   val cd = ContactDetails("c1", "c2", "c3", "c4", "c5")
   val confirmCd = ConfirmedContactDetails(cd)
-  val ec = "council_tax"
+  val councilTax = "council_tax"
```
Comment (valydia) [inline test/uk/gov/hmrc/valuationofficeagencycontactfrontend/views/ConfirmationViewSpec.scala:29]: Good catch 👍

**Pull request merged**